pkg/daemon/ceph/client/mirror.go

```diff
@@ -376,6 +376,12 @@ func EnableRBDRadosNamespaceMirroring(context *clusterd.Context, clusterInfo *Cl
 		return errors.Errorf("ceph version %q does not support mirroring in rados namespace %q with --remote-namespace flag, supported version are v20 and above.", clusterInfo.CephVersion.String(), poolAndRadosNamespaceName)
 	}
 
+	implicitnsKey := "<implicit>"
```

Comment:
Let's just spell out `namespace`, for readability
```suggestion
	implicitNamespaceKey := "<implicit>"
```

Reply:
got it